### DISCUSSION TITLE
Mageplaza Instagram Feed code is not compitable with breeze theme

### DISCUSSION THREAD

**Discussion by Chandu9012** [breezefront/community#166] [1. Help]
I am using Mageplaza InstagramFeed extension and it is working fine on login page where login page excluded from breeze. I need Instagram Feed on homepage and category page but there are not working the nstagram feed because those pages are not excluded from breeze. I have installed breeze theme extension as well.

I need to make below code OR Mageplaza instagram feed code to compatible with breeze. Please help me.

```
<script type="text/x-magento-init"> 
{
    "#mpinstagramfeed-photos-": {
        "mageplaza/instagram": {
            "id": "",
            "url": "getUrl('mp_instagram/feed/get') ?>",
            "count": "",
            "layout": "",
            "items_slider": "",
            "show_caption": "",
            "show_popup": "",
            "show_video": ""
        }
    }
}
</script>
```

**Comment by vovayatsyuk** [comment]
If "mageplaza/instagram" is a simple widget, you can register its js file using layout update XML: https://breezefront.com/docs/components#component-registration

Just added a new article to our docs: https://breezefront.com/docs/integration-example
